src/models/patient.ts

```diff
@@ -28,9 +35,26 @@ export const genPatientDetails = ({
     euNam.gnt = convertStandardizationName(firstName);
   }
 
+  const validateDateOfBirth = dateOfBirth.split('T')[0]; // take only date value if ISO-8601 format
+  if (
+    !dayjs(
+      validateDateOfBirth,
+      ["YYYY", "YYYY-MM", "YYYY-MM-DD"],
+      true
+    ).isValid()
+  ) {
+    throw new Error(
+      `Invalid dateOfBirth (${dateOfBirth}). Should be YYYY-MM-DD or YYYY-MM or YYYY or ISO-8601 format`
+    );
+  }
+
+  const dob = dayjs(dateOfBirth, ["YYYY", "YYYY-MM", "YYYY-MM-DD"], true).isValid()
+  ? dateOfBirth
+  : dayjs.utc(dateOfBirth).format("YYYY-MM-DD");
```

Comment:
no need timezone converter for this UTC date. both input date and eu offline qr dob date need to be UTC date. Shouldn't need to use dayjs timezone plugin to convert.
Note : EU required date format is like this;
<img width="631" alt="Screenshot 2022-05-15 at 9 21 07 PM" src="https://user-images.githubusercontent.com/10098074/168475000-f9b64431-9845-45b7-98aa-c94c2b9b52f6.png">